packages/dataviews/src/components/dataviews-pagination/style.scss

```diff
@@ -12,10 +12,12 @@
 }
 
 .dataviews-pagination__page-selection {
-	font-size: 11px;
-	text-transform: uppercase;
-	font-weight: 500;
-	color: $gray-900;
+	.components-select-control__input {
+		&:hover,
+		&:focus {
+			color: $gray-900;
```

Comment:
cc @mirka when using the `minimal` variant of `SelectControl`, some wp-admin styles bleed through affecting the hover/focus color. I wonder if we should fix that in the component?

Reply:
Agreed! Fix at #63855